model/pipeline/src/main/resources/org/apache/beam/model/common_urns.md

```diff
@@ -0,0 +1,122 @@
+<!--
+
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Apache Beam URNs
+
+This file serves as a central place to enumerate and document the various
```

Comment:
<!--new_thread; commit:8d3085bf4a646efa65e3a40f0c4528297fb54694; resolved:0-->
Maybe YAML would allow easier parsing and machine-readable association of metadata and commentary with the URNs? This file already suggests a `comment` and `payload` field for general description and a machine-readable spec for the payload.